NDB.Covid19/NDB.Covid19/ViewModels/DailyNumbersViewModel.cs

```diff
@@ -65,6 +66,17 @@ static DailyNumbersViewModel()
             ? string.Format(DAILY_NUMBERS_SUBSUBHEADER, $"{DateUtils.GetDateFromDateTime(LastUpdateDownloadsNumbersDateTime, "m")}", $"{DateUtils.GetDateFromDateTime(LastUpdateDownloadsNumbersDateTime, "t")}")
             : "";
 
+        public static async void RequestFHIDataUpdate(Action onFinished = null)
+        {
+            // Update only if the date of the last update is different than today
+            // to limit the requests
+            if (FHILastUpdateDateTime.Date != SystemTime.Now().ToLocalTime().Date)
```

Comment:
Won't it lead to some issues? Consider the following, FHI updates stats at 13.00 Norwegian time. If you do this check at some time before that, you will update `FHILastUpdateDateTime` but not necessarily fetch new data. But then until the end of the day you won't fetch new data if returned from background. 
Am I missing something here or overthinking it?

Reply:
I cannot see issues here. FHILastUpdateDateTime is the date of the last fetch on the server side, not on device (unless the behavior has changed for Norwegian app) so it is updated only once per day. So this check should just limit the number of requests.